lib/pdk/cli/module/generate.rb

```diff
@@ -0,0 +1,40 @@
+require 'tty-prompt'
+
+module PDK::CLI
+  @module_generate_cmd = @module_cmd.define_command do
+    name 'generate'
+    usage _('generate [options] <module_name>')
+    summary _('This command is now \'pdk new module\'.')
+
+    PDK::CLI.template_url_option(self)
+
+    flag nil, 'skip-interview', _('When specified, skips interactive querying of metadata.')
+
+    run do |opts, args, _cmd|
+      require 'pdk/generators/module'
+
+      module_name = args[0]
+
+      if module_name.nil? || module_name.empty?
+        puts command.help
+        exit 1
+      end
+
+      PDK.logger.info(_('New modules are created using the ‘pdk new module’ command.'))
+      prompt = TTY::Prompt.new(help_color: :cyan)
```

Comment:
This is a different colour to the help text in the rest of the prompts (probably a lot more readable than "bright black", not to mention a lot less painful to try to imagine). Should we adjust the help text in the interview to also use cyan for consistency?

Reply:
@rodjek the interview also uses cyan as the `help_color`. The TTY::Prompt is just set up and passed into the interview from the generator.

https://github.com/puppetlabs/pdk/blob/master/lib/pdk/generators/module.rb#L217